[53505130] Add the Move, Mood and Motivation questionnaire to the SDV project (#3092)
compsy/u-can-act: README.md

}
```

## Developing new questionnaires

> When creating a new questionnaire(or questionnaires), these are the steps you should follow:

Reviewer: ```suggestion
When creating a new questionnaire (or questionnaires), these are the steps you should follow:
```

1. Define the questions of the questionnaire 
2. Create a protocol
3. Define the measurements of that protocol

### Defining the questionnaire

> To define which questions compose the new questionnaire, you should create a new .rb file under `projects/sport-data-valley/seeds/questionnaires/`.

Reviewer: ```suggestion
To define which questions compose the new questionnaire, you should create a new .rb file under `projects/<project-name>/seeds/questionnaires/`.
```

> In this file, you should create a new `Questionnaire` object on the DB, the define which questions appear in the questionnaire, and finally save that object.\

Reviewer: ```suggestion
In this file, you should create a new `Questionnaire` object on the DB, the define which questions appear in the questionnaire, and finally save that object.
```
